libs/wingsdk/src/std/bool.ts

```diff
@@ -7,7 +7,7 @@ export class Boolean {
   /**
    * Parse a boolean from Json.
    *
-   * @macro ((args) => { if (typeof args !== "boolean") {throw new Error(`unable to parse ${typeof args} "${args}" as a boolean`)}; return JSON.parse(JSON.stringify(args)) })($args$)
+   * @macro ((args) => { if (typeof args !== "boolean") {throw new Error("unable to parse" + " " + typeof args + " " + args + " as a boolean")}; return JSON.parse(JSON.stringify(args)) })($args$)
```

Comment:
I think you could just convert to single quotes and avoid string concatenation via + sign:
```suggestion
   * @macro ((args) => { if (typeof args !== "boolean") {throw new Error('unable to parse ${typeof args} ${args} as a boolean')}; return JSON.parse(JSON.stringify(args)) })($args$)
```

Reply:
Id love to get rid of the concatenation with `+` but this would not do string interpolation outside of an assert method though.
IE: 

```js
let j = Json {a: "cool"};

assert(bool.from_json(j.get("a")));  // <==== this would give: preflight error: unable to parse string cool as a boolean
```
```js
let j = Json {a: "cool"};

bool.from_json(j.get("a"));  // <==== this would give: preflight error: unable to parse ${typeof args} ${args} as a boolean
```

Reply:
Got it, thanks for clarifying.